index.d.ts

```diff
@@ -2,33 +2,29 @@ export { } from "./jsx";
 
 export as namespace skate;
 
-export interface HasProps<El extends Component> {
-  props: { [key: string]: skate.PropOptions<El, any>; };
-}
-
-export interface OnUpdatedCallback {
```

Comment:
I understand, but our users are extending the skate Component and not implementing an interface.
These are methods that exist on the Component class.
Oner can choose to, or not to, override them.
But if overridden one must call super.